src/client/pages/OfferNew/Tabs/Tabs.tsx

```diff
@@ -31,26 +43,45 @@ type Props = {
   }[]
   onChange: (id: string) => void
 }
-export const Tabs: React.FC<Props> = ({ items }) => {
+type Item = {
+  id: string
+  name: string
+  content: React.ReactNode
+}
+
+export const Tabs: React.FC<Props> = ({ items, onChange }) => {
   const [selected, setSelected] = useState(items[0])
+
+  const handleClick = (item: Item) => {
+    setSelected(item)
+    onChange(item.id)
+  }
+  useEffect(() => {
+    setSelected(items[0])
+  }, [items])
```

Comment:
Does this do what we expect? Perhaps we should be a bit more conservative:

```suggestion
  useEffect(() => {
    setSelected((selectedItem) => {
      const matchingItem = items.find((item) => item.id === selectedItem.id)
      return matchingItem ?? items[0]
    })
  }, [items])
```